app/controllers/concerns/curation_concerns/curation_concern_controller.rb

```diff
@@ -105,6 +105,11 @@ def file_manager
       presenter
     end
 
+    def inspect_work
+      # TODO: How to hide this action behind admin role. It currently depends on user ability for work
```

Comment:
@jcoyne The TODO above (constraining the inspect_work view to the admin role) has been completed and pushed. The controller test was changed accordingly to expect admin ability.